main.js

```diff
@@ -1,78 +1,103 @@
-/* eslint-disable no-use-before-define */
-// eslint-disable-next-line max-classes-per-file
-const listOfBooks = document.querySelector('.book-list');
-
-class Book {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-    this.bookid = Math.random().toFixed(1);
-  }
-}
+// Array Constructor
 
-class StoreBook {
-  constructor() {
-    // Array of book items
-    this.BookData = [];
-  }
+const booksObject = {};
 
-  // New books
-  addBook = (newbook) => {
-    this.BookData.push(newbook);
-    localStorage.setItem('AllBooks', JSON.stringify(this.BookData));
-    DisplayBooks(newbook);
-  };
-
-  // Remove book from the list
-  removeBook = (bookid) => {
-    const deleteBook = document.getElementById(bookid);
-    deleteBook.remove();
-    this.BookData = this.BookData.filter((x) => x.bookid !== bookid);
-    localStorage.setItem('AllBooks', JSON.stringify(this.BookData));
-  };
-}
-
-const savebook = new StoreBook();
-// Get input value
-const getformInput = () => {
-  const title = document.querySelector('.title');
-  const author = document.querySelector('.author');
-  const insertbook = new Book(title.value, author.value);
-  return insertbook;
-};
+// Storage Functions
 
-// Display the list of books on the web page
-let DisplayBooks = (index) => {
-  let bgcolor = '';
-  if (savebook.BookData.indexOf(index) % 2 !== 0) {
-    bgcolor = '#fff';
+const getBooks = () => {
+  let books;
+  if (localStorage.getItem('books') === null) {
+    books = [];
   } else {
-    bgcolor = '#ff99';
+    books = JSON.parse(localStorage.getItem('books'));
   }
-  const newBook = document.createElement('div');
-  newBook.classList.add('book-item');
-  newBook.classList.add(bgcolor);
-  newBook.setAttribute('id', index.bookid); newBook.innerHTML = `
- <p>${index.title} <br> written by ${index.author}</p> <hr>`;
-  const removeBook = document.createElement('button');
-  removeBook.innerHTML = 'Remove';
-  removeBook.addEventListener('click', () => savebook.removeBook(index.bookid));
-  newBook.appendChild(removeBook);
-  listOfBooks.appendChild(newBook);
+  return books;
 };
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/226551547-efbc1485-b835-4be9-8083-7ee90fb6d720.png)

**[OPTIONAL]**

- [ ] It would be nice if you make all your commit messages meaningful and capitalize because some of them are not capitalized and some are not meaningful enough which is making your pull request unprofessional 😨